hexnumber/hexnumber.py

```diff
@@ -0,0 +1,99 @@
+import sys
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 renamed some functions 
 Mistakes: 
 Dude...it is not even in right format.